### PR TITLE
[I18N] Update Localizable.strings

### PR DESCRIPTION
**Pull request by TheMythPT**
Did you missed me? 😛
Translated the remaining strings. 😁
If there's some strings that aren't translated, it's because i don't find a proper translation to it.
Basically, if you don't see `// TODO` in the front of the string, consider it ready for production.
🎉 

@RocketChat/ios

### CODE REVIEW

File: Rocket.Chat/Resources/pt-PT.lproj/Localizable.strings [hunk 1]
```diff
@@ -155,11 +155,11 @@
...
 "subscriptions.list.sent_an_attachment" = "enviou um ficheiro";
 "subscriptions.list.date.yesterday" = "Ontem";
-"subscriptions.list.actions.read" = "Lido"; // TODO
-"subscriptions.list.actions.unread" = "Não lido"; // TODO
-"subscriptions.list.actions.favorite" = "Favorite"; // TODO
-"subscriptions.list.actions.unfavorite" = "Unfavorite"; // TODO
-"subscriptions.list.actions.hide" = "Fechar"; // TODO
+"subscriptions.list.actions.read" = "Lido";
+"subscriptions.list.actions.unread" = "Não lido";
+"subscriptions.list.actions.favorite" = "Marcar como favorito";
```
Comment (rafaelks) [inline Rocket.Chat/Resources/pt-PT.lproj/Localizable.strings]: Do you have a "shorter" way to say that? The UI for this button is very small, preferable just one word.

In Portuguese we are using "Favoritar"

Comment (TheMythPT) [reply Rocket.Chat/Resources/pt-PT.lproj/Localizable.strings]: Hope the new PR is more within the idea

File: Rocket.Chat/Resources/pt-PT.lproj/Localizable.strings [hunk 2]
```diff
@@ -269,10 +269,10 @@
 "chat.drawing.settings.color.others" = "Escolha uma cor abaixo:";
 
 // Chat Typing
-"chat.users_are_typing" = "%@ are typing..."; // TODO
-"chat.user_is_typing" = "%@ is typing..."; // TODO
-"chat.typing" = "typing..."; // TODO
-"chat.several_typing" = "several users are typing..."; // TODO
+"chat.users_are_typing" = "%@ estão escrevendo...";
+"chat.user_is_typing" = "%@ está escrevendo...";
+"chat.typing" = "escrvendo...";
```
Comment (rafaelks) [inline Rocket.Chat/Resources/pt-PT.lproj/Localizable.strings]: Typo?

Comment (TheMythPT) [reply Rocket.Chat/Resources/pt-PT.lproj/Localizable.strings]: those happen :stuck_out_tongue: 
will fix it now

**Pull request merged**